exp/views/study.py

```diff
@@ -330,6 +332,23 @@ class StudyResponsesList(LoginRequiredMixin, PermissionRequiredMixin, generic.De
     permission_required = 'studies.can_view_study_responses'
     raise_exception = True
 
+    def get_context_data(self, **kwargs):
+        """
+        In addition to the study, adds several items to the context dictionary.  Study results
+        are paginated.
+        """
+        context = super().get_context_data(**kwargs)
+        orderby = self.request.GET.get('sort', 'id')
+        page = self.request.GET.get('page', None)
+        study = context['study']
+        responses = study.responses.filter(completed=True).order_by(orderby) if orderby else study.responses.all()
```

Comment:
If `orderby` then filter by `completed` else give them everything? Orderby has a default of `id` so unless it's set to something falsey it's always going to be truthy. Right?